packages/server/modules/core/tests/integration/scanTable.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { describe } from 'mocha'
+import { scanTableFactory } from '@/modules/core/helpers/scanTable'
+import { db } from '@/db/knex'
+import { UserRecord } from '@/modules/core/helpers/types'
+import { Users } from '@/modules/core/dbSchema'
+import { expect } from 'chai'
+import crs from 'crypto-random-string'
+import {
+  createRandomEmail,
+  createRandomPassword
+} from '@/modules/core/helpers/testHelpers'
+
+const scanTable = scanTableFactory<UserRecord>({ db })
+
+describe('Helpers scanTable @core', () => {
+  before(async () => {
+    await db(Users.name).delete()
```

Comment:
We have utils for truncating tables: `import { truncateTables } from '@/test/hooks'`